easybuild/easyconfigs/m/mawk/mawk-1.3.4-20141206-GCC-4.9.2.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'mawk'
+version = '1.3.4-20141206'
+
+homepage = 'http://invisible-island.net/mawk/mawk.html'
+description = """mawk is an interpreter for the AWK Programming Language."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['ftp://invisible-island.net/mawk']
+
+sanity_check_paths = {
+    'files': ['bin/mawk'],
+    'dirs': ["bin"]
```

Comment:
The sanity check step will first check all files/dirs, and only then start issuing errors.

So, if `bin/mawk` is missing in this case, it will both report `bin/mawk` to be missing and not finding a non-empty `bin` directory (unless something else is in there).